cypress/support/e2e.js

```diff
@@ -0,0 +1,20 @@
+// ***********************************************************
```

Comment:
This is also an unnecessary file, unless you're actually adding some config here. Rather remove both the support/ and fixtures/ folders and edit the cypress.config.js to include `supportFile: false`

Reply:
Fixed, nice catch :D